### PR TITLE
[BPK-4276] Simplify page structure for some pages

### PR DESCRIPTION
**Pull request by shaundon**
Simplified the folder structure for a few pages into a new format.

From this:

```
pages/
    AndroidBadgePage/
        AndroidBadgePage.js
        index.js
    BadgePage/
        BadgePage.js
        index.js
    IOSBadgePage/
        IOSBadgePage.js
        index.js
    NativeBadgePage/
        NativeBadgePage.js
        index.js
    WebBadgePage/
        WebBadgePage.js
        index.js
```

..with this:

```
pages/
    BadgePage/
        BadgePage.js
        AndroidBadgePage.js
        IOSBadgePage.js
        NativeBadgePage.js
        WebBadgePage.js
        index.js
```

This is a very boring and tedious task, so I only did it for a few pages. Maybe we should do it in batches to make it easier.

### CODE REVIEW

File: docs/src/constants/generated/routes.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ import MarkdownPage from '../../components/MarkdownPage';
 export const using = [{path: "/using/backpack-react-scripts", component: () => (<MarkdownPage title="Backpack React Scripts" subtitle="Create React + Backpack web apps with no build configuration." content={`
 ## TL;DR
 
-**The currently supported version of CRA by backpack-react-scripts is up to v3.4.1. Versions above this will not work.**
+**The currently supported version of CRA by backpack-react-scripts is up to \`v3.4.1\`. Versions above this will not work.**
```
Comment (shaundon) [inline docs/src/constants/generated/routes.js:26]: These weren't checked in last time 🙈

**Review by georgegillams — APPROVED** [review]
